lib/fog/azurerm/models/dns/record_set.rb

```diff
@@ -50,6 +61,23 @@ def destroy
         def get_records(resource_group, name, zone_name, record_type)
           service.get_records_from_record_set(resource_group, name, zone_name, record_type)
         end
+
+        def update_ttl(ttl)
+          record_set = service.create_or_update_record_set(resource_group, name, zone_name, records, type.split('/').last, ttl)
+          merge_attributes(Fog::DNS::AzureRM::RecordSet.parse(record_set))
+        end
+
+        def add_a_type_record(record)
```

Comment:
Its a single record which is adding and remove from the list of records, user provide a single record.